lib/config_cat/fetch_policy.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ConfigCat.FetchPolicy do
```

Comment:
Please create 3 different files for the different polling modes if possible:
auto_polling_cache_policy.ex
lazy_loading_cache_policy.ex
manual_polling_cache_policy.ex

Reply:
Extracted to #15